experimental/PlaneCurve/ParaPlaneCurves.jl

```diff
@@ -140,6 +258,40 @@ function rat_normal_curve_It_Proj_Odd(C::ProjCurve)
     return gens(ideal(R, J))
 end
 
+@doc Markdown.doc"""
+    rat_normal_curve_It_Proj_Even(C::ProjCurve)
```

Comment:
Why is it spelled like this, with a few upper case letter in there? What does "It" mean?